Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -117,6 +118,7 @@ extension LoginViewController: EndpointDelegate{
             //Notify user of unauthorized reload the view on the main thread
             DispatchQueue.main.async {
                 self.reloadViewWithUsernameAndInstanceSaved()
+                self.viewDidLoad()
```

Comment:
Why are we calling viewDidLoad? I don't think that we are supposed to call that method directly. I think it is called only after a view is loaded in, in the lifecycle of a view. Aside from that, what in viewDidLoad do we need to call from here? It doesn't seem to me like calling this method would change anything.